packages/react-components/src/components/RichTextEditor/RichTextSendBox.tsx

```diff
@@ -392,9 +386,9 @@ export const RichTextSendBox = (props: RichTextSendBoxProps): JSX.Element => {
     if (
       hasContent ||
       /* @conditional-compile-remove(file-sharing-acs) */ isAttachmentUploadCompleted(attachments) ||
-      /* @conditional-compile-remove(rich-text-editor-image-upload) */ isAttachmentUploadCompleted(
-        inlineImagesWithProgress
-      )
+      // hasContent returns false if the content only contains img tags.
```

Comment:
Should hasContent return false in this case, is it possible to update that method so that it returns true when we have images?